src/components/legacy/icons/form-icons/clear.tsx

```diff
@@ -10,13 +10,13 @@ export const Clear = () => (
   >
     <path
       d="M1 1L11 11"
-      stroke="#ffffff"
+      stroke="currentColor"
```

Comment:
The white colors of the close x to remove a file was almost not readable
![Screen Shot 2022-02-04 at 15 59 57](https://user-images.githubusercontent.com/1701755/152551370-71e962af-912b-4b50-b4fc-c2df1a40667e.png)
![Screen Shot 2022-02-04 at 16 01 14](https://user-images.githubusercontent.com/1701755/152551430-60e5a517-28d1-4d03-aa01-e59363857946.png)